Mergin/projects_manager.py

```diff
@@ -235,19 +235,23 @@ def sync_project(self, project_dir, project_name=None):
             return
 
         if dlg.pull_conflicts:
-            msg = "Following conflicts between local and server version found: \n\n"
+            msg = "Sync aborted as there are conflicting changes in local and server versions.\n" +
```

Comment:
Saying that sync was aborted is probably not entirely correct, as pull finished successfully, it "just" created some conflicted copies. I would suggest to use the following wording:
`
We have detected conflicting changes between your local copy and the server that could not be resolved automatically. It is recommended to manually reconcile your local changes (which have been moved to conflicted copies) before running sync again.
`

Reply:
Done.